device/bootloader/Source/main.c

```diff
@@ -409,6 +409,11 @@ int main(void) {
     NRF_DPPIC_NS->CHENSET = (DPPIC_CHENSET_CH0_Enabled << DPPIC_CHENSET_CH0_Pos);
     NRF_DPPIC_S->CHENSET = (DPPIC_CHENSET_CH0_Enabled << DPPIC_CHENSET_CH0_Pos);
 
+    // Allow the network core to access GPIO port 1
+    NRF_SPU_S->GPIOPORT[1].PERM = 0;
```

Comment:
By doing this you set all pins on port 1 as non secure while you only want P1.04.
```suggestion
    NRF_SPU_S->GPIOPORT[1].PERM = ~(1 << 4);
```
(Untested, but the idea is to only set the bit corresponding to p1.04 as non secure, e.g 0, while keeping all others to 1)

Reply:
Thanks, just updated this. It seems to work, but the LED light is very faint, and when the node loses the gateway, it blinks a few times which is weird.